reStream.sh

```diff
@@ -28,6 +29,27 @@ while [ $# -gt 0 ]; do
             shift
             shift
             ;;
+        -w | --webcam)
+            webcam=true
+            format="v4l2"
+
+            # check if there is a modprobed v4l2 loopback device
+            # use the first cam as default if there is no output_path already
+            cam_path=$(v4l2-ctl --list-devices \
+                | sed -n '/^[^\s]\+platform:v4l2loopback/{n;s/\s*//g;p;q}')
+
+            # fail if there is no such device
+            if [ -e "$cam_path" ]; then
+                if [ "$output_path" = "-" ]; then
+                    output_path="$cam_path"
+                fi
+            else
+                echo "Could not find a video loopback device, did you"
+                echo "sudo modprobe v4l2loopback"
```

Comment:
This will probably need some explanation in the README what needs to be installed and loaded. Would you like to do that, or should I?

Reply:
We need v4l-utils and v4l2loopback-dkms but I do not know what is needed in ubuntu

Reply:
On Ubuntu it is 
```
sudo apt install v4l2loopback-utils v4l2loopback-dkms
```
on arch it was
```
yay -S v4l-utils v4l2loopback-dkms
```

Reply:
@ax3l are you running Ubuntu by any chance? If so, could you document the installation steps needed to get v4l2 working?

Reply:
See: https://github.com/rien/reStream/pull/18/commits/9e9fbbc77185e4abf4899d28092fbf819a59193a